### DISCUSSION TITLE
Help configuration

### DISCUSSION THREAD

**Discussion by FranciscoPalomares** [globaleaks/globaleaks-whistleblowing-software#3658] [Q&A]
Hello,
Some questions regarding the configuration: 

How to add the long id of the report received by the user to the report table of the administration portal ?

On the other hand, is there any way to add a contact person in the report and have him/her receive follow-up information about the report?

Thank you very much

**Comment by evilaliv3** [comment]
Thank you for your question @FranciscoPalomares 

May i ask you the following clarifications?

1. What do you mean with "long id of the report"? 
2. Wat do you many with adding a contact person in the report; who should add it and which is the role of these contact persons in the process?

**Comment by FranciscoPalomares** [comment]
Hi,
1. I mean the tracking id that the user receives when adding a report and has to insert it in order to see the report tracking.
2. The idea is that the user who posts the report, the user can add a contact email, and when the recipient adds a comment to the report, this will be notified by email.
3. Thanks

**Comment by evilaliv3** [comment]
Thank you @FranciscoPalomares 

Regarding 1. the receipt is a secret of the whistleblower that is not saved on the system and should be known only to the whistleblower. it is basically an unknown password and globaleaks do not stored it and the whistleblower shall never expose it to anyone. As different thing we are instead implementing that the interface will show the internal UUID4 assigned to the report; trhis information will be then visible to whistleblowers, administrators and recipients.
